cmd/strelaysrv/main.go

```diff
@@ -186,6 +186,7 @@ func main() {
 	}
 
 	wrapper := config.Wrap("config", config.New(id), id, events.NoopLogger)
+	go wrapper.Serve(context.TODO())
```

Comment:
We don't do fancy service magic in strelaysrv historically and we just need this running in order to do the modify below, so `go`ing here instead of pulling in suture.